pyproject.toml

```diff
@@ -43,9 +43,13 @@ doc = [
 	# local
 ]
 
+ruff = [
+	# Get requires-python from pyproject.toml automatically astral-sh/ruff#16319
+	"ruff >= 0.11.0",
+]
+
 check = [
 	"pytest-checkdocs >= 2.4",
-	"pytest-ruff >= 0.2.1; sys_platform != 'cygwin'",
```

Comment:
Probably we want to revert the changes in this file. Ruff won't run on cygwin, and we definitely want `pytest-ruff` so that ruff runs under pytest. If `ruff >= 0.11` is needed, we can just wait for that to become a de facto minimum.

Reply:
Part of the proposition here is to *not* run under pytest, nor use pytest-ruff.
And no cygwin run installs ruff.